.github/workflows/examples.yml

```diff
@@ -38,6 +38,14 @@ jobs:
     timeout-minutes: 120
     if: github.event_name != 'schedule' || github.repository == 'mlflow/mlflow'
     steps:
+      - name: Free Ubuntu Disk Space
+        uses: jlumbroso/free-disk-space@main
```

Comment:
It's actually much more:

``` shell
overall:
********************************************************************************
=> Saved 22GiB
********************************************************************************
```

Reply:
Historic disk usage with `df -h`: 
![Screenshot 2023-06-05 at 1 55 51 PM](https://github.com/mlflow/mlflow/assets/39283302/b800238b-9b58-45ff-beb8-6647cdb9ab20)

Disk usage after changes in this PR:
![Screenshot 2023-06-05 at 1 56 04 PM](https://github.com/mlflow/mlflow/assets/39283302/3b75b68d-db42-45db-bd35-092d4579b454)

Reply:
Running `sudo rm -rf "$AGENT_TOOLSDIRECTORY"` might be faster.

Reply:
Do you remember how much space that saved? Based on the discussions by the creator of that action runner script, it seemed like that was only a few GB? 
With some of these new flavors and the size of their models, I think the large-scale approach like this might prevent us some headaches

Reply:
<img width="522" alt="image" src="https://github.com/mlflow/mlflow/assets/17039389/d21ad560-5bc4-4105-ad9b-db00c7e8f4df">

It can free up 9GB with 2 seconds.

Reply:
```
sudo rm -rf "$AGENT_TOOLSDIRECTORY"
sudo rm -rf /usr/share/dotnet
sudo rm -rf /usr/local/lib/android
```

<img width="522" alt="image" src="https://github.com/mlflow/mlflow/assets/17039389/a1bb9ec9-9454-4abd-9f01-10eb6e520d86">

Reply:
testing this in https://github.com/mlflow/mlflow/pull/8610

Reply:
@BenWilson2 Can we just run this command? This is much faster.